diesel_cli/src/config.rs

```diff
@@ -381,6 +381,15 @@ impl Config {
                 let derives = derives.cloned().collect();
                 config.custom_type_derives = Some(derives);
             }
+
+            if let Some(domains) = matches.get_many::<String>("domains-as-custom-types") {
+                let regexes: Vec<String> = domains.cloned().collect();
```

Comment:
Technically we don't need that collect, right?

Possibility even the cloned can be skipped given that the regex takes a &str. 

That's a minor nit, it shouldn't matter much, so feel free to ignore.

Reply:
Absolutely wouldn't want to ignore, here we go b213b2e44ae3942e172d109e6a90712804062dc6